notifications_utils/sanitise_text.py

```diff
@@ -122,19 +122,15 @@ def is_arabic(cls, value):
     def is_punjabi(cls, value):
         # Gukmukhi script or Shahmukhi script
 
-        if regex.search(r"[\u0A00-\u0A7F]+", value):
-            return True
-        elif regex.search(r"[\u0600-\u06FF]+", value):
-            return True
-        elif regex.search(r"[\u0750-\u077F]+", value):
-            return True
-        elif regex.search(r"[\u08A0-\u08FF]+", value):
-            return True
-        elif regex.search(r"[\uFB50-\uFDFF]+", value):
-            return True
-        elif regex.search(r"[\uFE70-\uFEFF]+", value):
-            return True
-        elif regex.search(r"[\u0900-\u097F]+", value):
+        if (
+            regex.search(r"[\u0A00-\u0A7F]+", value)
+            or regex.search(r"[\u0600-\u06FF]+", value)
+            or regex.search(r"[\u0750-\u077F]+", value)
+            or regex.search(r"[\u08A0-\u08FF]+", value)
+            or regex.search(r"[\uFB50-\uFDFF]+", value)
+            or regex.search(r"[\uFE70-\uFEFF]+", value)
+            or regex.search(r"[\u0900-\u097F]+", value)
+        ):
```

Comment:
When I first saw this code block I was like "oooh, oooh, a chance to use a [`match` statement](https://docs.python.org/3/tutorial/controlflow.html#match-statements)!"  But as I look closer, I'm not sure that's actually worth it since it's just a check for any of these conditions being `True`.

Something is telling me there might be a better way to handle this, but this is also super readable and clean as is. So consider this just a note for a potential future thought exercise on using a `match` statement when there's a bunch of `if`/`elif`/`else` statements! 🙂